stages/9-openebs-install-check/openebs-control-plane-test.md

```diff
@@ -0,0 +1,174 @@
+# openebs-control-plane-test
+
+---
+tcid: iuoi03
+
+name: "Install OpenEBS Control plane components on a specific set of nodes"
+sidebar_label: openebs-installation-test-litmus
+---
+------
+
+## Experiment Metadata
+
+<table>
+  <tr>
+    <th> Type </th>
+    <th> Description </th>
+    <th> Tested K8s Platform </th>
+  </tr>
+  <tr>
+    <td> Openebs Installation </td>
+    <td> Openebs Control plane Installation on specific set of nodes</td>
+    <td> GKE </td>
+  </tr>
+</table>
+
+## Prerequisites
+
+- Along with k8s, Litmus should be installed in the cluster.
+- Every component of DOP cluster should be healthy and running.
+- Ensure that the `openebs-control-plane-test` resource is available in the cluster.
+
+## Entry Criteria
+
+- Kubernetes nodes are up and running with no control plane labels.
+
+## Exit Criteria
+
+- Kubernetes nodes are up and running with control plane labels as true. 
+
+## Details
+
+- In this test case, we have to install openebs on a particular node. It can be done by labeling the nodes before installing the openebs on it. So basically we will test that the labeling feature is working fine with respective of control plane components. Control plane components include all the openebs components expect NDM daemonset.
+
+- The tasks involved for creating the test case will be - we will labels nodes as shown in the table.
+
+OpenEBS components | Node-1 | Node-2 | Node-3
+-- | -- | -- | --
+Control plane components | True | Flase | False
+Data plane components | False | True | True
```

Comment:
Can you please use the below format to create table here.
ex.
```
<table>
  <tr>
    <th> </th>
    <td> </td>
  </tr>
</table>
```

Reply:
It will look messy ... Anyways its Two row tabel

Reply:
Actually It may cause conflict at the time of readme updation Thatswhy I am asking you to change the table format